### PR TITLE
Logger implementation in profile_builder

### PR DESCRIPTION
**Pull request by ChrisWallace2020**
Uses `logger.info` instead of prints and doesn't use tqdm if the user specifies a verbosity level above INFO.

Corresponding tests added.

Addresses #265

### CODE REVIEW

File: dataprofiler/tests/test_dp_logging.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def setUpClass(cls):
         dp_logging._dp_logger = None
 
     @classmethod
-    def tearDown(cls):
+    def tearDownClass(cls):
```
Comment (ChrisWallace2020) [inline dataprofiler/tests/test_dp_logging.py:18]: Fix I forgot to include in previous PR

**Comment by ChrisWallace2020** [comment]
WIP until Jeremy and I come up with a more elegant way to handle TQDM with logging.

**Comment by ChrisWallace2020** [comment]
Will save more elegant TQDM solution for future PR

File: dataprofiler/tests/profilers/test_profile_builder.py [hunk 2]
```diff
@@ -1360,6 +1361,48 @@ def test_diff_with_different_schema(self, *mocks):
...
+
+        # Ensure tqdm printed progress bar
+        self.assertIn('#' * 10, mock_stderr.getvalue())
+
+        # Clear stderr
+        mock_stderr.seek(0)
+        mock_stderr.truncate(0)
+
+        # Now tqdm shouldn't be printed
+        dp.dp_logging.set_verbosity(logging.WARNING)
```
Comment (JGSweets) [inline dataprofiler/tests/profilers/test_profile_builder.py:1398]: should this just be `dp.set_verbosity`?

Comment (ChrisWallace2020) [reply dataprofiler/tests/profilers/test_profile_builder.py]: Added.

**Review by JGSweets — APPROVED** [review]

**Review by AnhTruong — APPROVED** [review]

**Pull request merged**